client.go

```diff
@@ -494,7 +494,11 @@ func (client *Client) worker() {
 // when client is nil. A channel is provided if it is important to check for a
 // send's success.
 func (client *Client) Capture(packet *Packet, captureTags map[string]string) (eventID string, ch chan error) {
+	ch = make(chan error, 1)
+
 	if client == nil {
+		// return a chan that always returns nil when the caller receives from it
```

Comment:
How so? The channel is closed, so it will always return nil. That is what `TestNilClient` is verifying. Did I screw something up?